develop/tutorials/articles/280-testing/02-arquillian-extension-liferay.markdown

```diff
@@ -1,611 +1,649 @@
+
+    		Assert.assertEquals(4, result);
+    	}
+
+    	@Inject
+    	private SampleService _sampleService;
+
+    }
+
+JUnit annotation `@RunWith(Arquillian.class)` marks the class for Arquillian to
+execute. 
+
+JUnit annotation `@Test` designates the `testAdd` method as a test method. It
+invokes a `SampleService` object's `add` method and asserts the result. 
+
+    final long result = _sampleService.add(1, 3);
+
+    Assert.assertEquals(4, result);
+
+The Liferay Arquillian Extension injects the class with the `SampleService`
+implementation (i.e., a `SampleServiceImpl` instance) via the `_sampleService`
+field. 
+
+    @Inject
+    private SampleService _sampleService;
+
+Arquillian packages the test class and resources as a Java archive (JAR file)
+using the class's `create` method. The method invokes the project's `jar`
+Gradle task to create the test JAR Arquillian executes. 
+
+    final File tempDir = Files.createTempDir();
+
+    final ProcessBuilder processBuilder = new ProcessBuilder(
+        "./gradlew", "jar", "-Pdir=" + tempDir.getAbsolutePath());
+
+    final Process process = processBuilder.start();
+
+    process.waitFor();
+
+    final File jarFile = new File(
+        tempDir.getAbsolutePath() +
+            "/com.liferay.arquillian.sample-1.0.0.jar");
+
+    return ShrinkWrap.createFromZipFile(JavaArchive.class, jarFile);
+
+Note: On Windows replace `gradlew` with `gradlew.bat` in the `create`
+method.
+
+Let's look at this test's dependencies. 
+
+#### Dependencies [](id=dependencies)
+
+The project's `build.gradle` file includes dependencies on Liferay's Arquillian container, JUnit, and Arquillian's JUnit container for this test:
+
+    testIntegrationCompile group: "com.liferay.arquillian", name: "com.liferay.arquillian.arquillian-container-liferay", version: "1.0.6"
+    testIntegrationCompile group: "junit", name: "junit", version: "4.12"
+    testIntegrationCompile group: "org.jboss.arquillian.junit", name: "arquillian-junit-container", version: "1.1.11.Final"
+
+Arquillian uses an XML configuration file called `arquillian.xml`. 
+
+#### Arquillian Configuration [](id=arquillian-configuration)
+
+The Arquillian configuration file `src/testIntegration/resources/arquillian.xml` specifies the test JAR location to the Arquillian engine. 
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <arquillian xmlns="http://jboss.org/schema/arquillian"
+    			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    			xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+        <!-- More conent here -->
+
+    	<engine>
+    		<property name="deploymentExportPath">build/deployments</property>
+    	</engine>
+    </arquillian>
+
+Enabling Java Management Extensions (JMX) on the application server's JVM allows
+tests that use Liferay's Arquillian Extension to deploy OSGi modules to
+@product@. 
+
+#### JMX Settings [](id=jmx-settings)
```

Comment:
The thing is, I have no problem running the Arquillian Blade Example and I don't see ARIES deployed on my DXP FP14 bundle.

So I'm not convinced installing AREIS is necessary.

What is its entry in GoGo schell's lb bundle listing?

How do you verify that ARIES is on DXP/Portal?

Thanks!